api/tests/unit/domain/usecases/start-writing-campaign-assessment-results-to-stream_test.js

```diff
@@ -551,5 +561,112 @@ describe('Unit | Domain | Use Cases | start-writing-campaign-assessment-results
         expect(csvLines[1]).to.equal(csvDataExpected);
       });
     });
+
+    context('when organization is SUP if add studentNumber to the CSV', () => {
+      let campaignWithoutIdPixLabel;
+      const organization = domainBuilder.buildOrganization({ type: 'SUP' });
+
+      beforeEach(() => {
+        const user = domainBuilder.buildUser({ firstName: '@Jean', lastName: '=Bono' });
+        const campaignParticipationResultData = {
+          id: 1,
+          isShared: false,
+          createdAt: new Date('2019-02-25T10:00:00Z'),
+          userId: 123,
+          participantFirstName: user.firstName,
+          participantLastName: user.lastName,
+          isCompleted: true,
+        };
+        user.memberships = [{ organization: { id:  organization.id } }];
+        userRepository.getWithMemberships.resolves(user);
+        findAssessmentResultDataByCampaignIdStub.resolves([campaignParticipationResultData]);
+        schoolingRegistrationRepositoryStub.resolves({ studentNumber: 'ie24052' });
+        campaignWithoutIdPixLabel = domainBuilder.buildCampaign.ofTypeAssessment({
+          name: 'CampaignName',
+          code: 'AZERTY123',
+          organizationId: organization.id,
+          idPixLabel: null,
+        });
+        campaignRepository.get.resolves(campaignWithoutIdPixLabel);
+        organizationRepository.get.resolves(organization);
+      });
+
+      it('should return the header and the data in CSV styles with all competence, domain and skills', async () => {
```

Comment:
ici on s'en fiche de vérifier toutes les en-têtes et toutes les data, seule la présence du numéro étudiant nous intéresse. Je propose de changer ce test.
```suggestion
      it('should return the header and the data with student number', async () => {
```